SPECS/blobfuse2/blobfuse2.spec

```diff
@@ -1,14 +1,14 @@
 %global debug_package %{nil}
 
 %define our_gopath %{_topdir}/.gopath
-%define preview_suffix preview.3	
+%define preview_suffix preview.4	
 %define blobfuse2_version 2.0.0	
 %define blobfuse2_health_monitor bfusemon
 
 Summary:        FUSE adapter - Azure Storage
 Name:           blobfuse2
 Version:        %{blobfuse2_version}.%{preview_suffix}
```

Comment:
Not a concern for this PR, but we're going to run into problems with this versioning scheme once the final version is released. The `.preview.X` portion causes the preview versions to be evaluated as a higher version than "2.0.0". Sorry that we didn't catch this when the package was first added.

Quick version comparison test in a 2.0 container:
```console
root [ / ]# rpm --eval '%[v"2.0.0.preview.4" < v"2.0.0"]'
0
root [ / ]# rpm --eval '%[v"2.0.0" < v"2.0.0.preview.4"]'
1
```

We can come up with a solution when the time comes, though.

Reply:
Yeah, when I originally added this package, I was having trouble with just making it a - for preview. Worst case we can do a GA release of Blobfuse2 as 2.0.1 or something and the version issue should go away